### DISCUSSION TITLE
Authorization always fails

### DISCUSSION THREAD

**Discussion by molodeztom** [alexkn/node-red-contrib-homeconnect#68] [Q&A]
I have set up an account at developer.home-connect.com. registered NodeRed... as an application and Code Grant Flow activated. 
ClientID and Secret entered in home-connect-auth node. When I start authorization a new browser tab opens where i have to log in with single key. Then it shows an "OK" on the page. When I go back to node red I got a message box:
requestToken failed: {
"error": invalid_client,
"error_description":"client has limited user list - user not assigned to client"
}
I dont get it why. Are there any special settings I need in home-connect such as One Time Token Mode (currently enabled). It shows Access "Private" maybe this is a problem?

**Comment by alexkn** [comment]
You have to set your "Home Connect User Account for Testing" in your developer account or application.

**Reply by molodeztom** [reply]
I have set Home Connect User Account to my email adress. This is also used in the Home Connect app. WhichNode-Red account does the home-connect-auth Node use?  I cant see any settings except Client ID and Secret. I am sorry hope my questions are not too stupid but I tried a lot of things already.

**Reply by Element2** [reply]
Have exactly the same problem.
Was this ever solved?
